sdk/Sdk.Generators/FunctionExecutor/FunctionExecutorGenerator.Emitter.cs

```diff
@@ -214,7 +214,9 @@ private static string EmitFastPath(ExecutableFunction function)
                 
                 if (function.IsObsolete)
                 {
-                    sb.Append("\n#pragma warning restore CS0618");
+                    // Environment.NewLine - Environment is banned for use in Analyzers
```

Comment:
The banning of NewLine is an oversight. 

https://github.com/Azure/azure-functions-dotnet-worker/blob/main/sdk/Sdk.Generators/Constants.cs#L12

https://github.com/dotnet/roslyn-analyzers/issues/6467